rel/config.exs

```diff
@@ -0,0 +1,34 @@
+use Mix.Releases.Config,
+    # This sets the default release built by `mix release`
+    default_release: :default,
+    # This sets the default environment used by `mix release`
+    default_environment: :dev
+
+# For a full list of config options for both releases
+# and environments, visit https://hexdocs.pm/distillery/configuration.html
+
+# You may define one or more environments in this file,
+# an environment's settings will override those of a release
+# when building in that environment, this combination of release
+# and environment configuration is called a profile
+
+environment :dev do
+  set dev_mode: true
+  set include_erts: false
+  set cookie: :"OFAba|8Y3zqSv}cw0Da]kOY*(6LusRdk{}4n4W.{g>/8|G|NewIwr)=@^&Qz2^rY"
+end
+
+environment :prod do
+  set include_erts: true
+  set include_src: false
+  set cookie: :"&uE.iv6/ry;]Js/(.6bCLK%b~Sa{sw>aSPrhWC43R5rUsRIxqqbC^MG.kZ!%ck*4"
```

Comment:
@jesseshieh not familiar with `Mix.Releases`, but I'm guessing we'll want to hide this in an env var before going live?

Reply:
The cookie here is actually overwritten my an env var called `${MY_COOKIE}` which is autogenerated for you. If you `gigalixir remote_console` and run `:erlang.get_cookie()` you should see a different value than the one here. But yeah, to reduce confusion, it might make sense to clear this one out.

Reply:
@jesseshieh dig. i'll clear this one tomorrow. thanks!